apps/rpc_client.py

```diff
@@ -69,34 +69,73 @@ def call(self, method, args=None):
 
     def sign_transaction(self, wallet_id, transaction):
```

Comment:
Please remove all unrelated code

Reply:
Still getting used to git - I reverted rpc_client to master and think I committed the change correctly